data/appstream/meson.build

```diff
@@ -0,0 +1,8 @@
+i18n.merge_file(
+  'gpaste-appdata',
+  input: 'org.gnome.GPaste.Ui.appdata.xml.in',
+  po_dir: '../../po',
+  output: 'org.gnome.GPaste.Ui.appdata.xml',
+  install: true,
+  install_dir: join_paths(get_option('datadir'), 'metainfo'),
```

Comment:
It was quite nice to have appstream-glib define this and not us with autotools. When they changed the install location, everything smoothly started using the new location with no change required. Is there a way to do such a thing there?

Reply:
> It was quite nice to have appstream-glib define this

Uhh...how did it do that?

Reply:
The package provides a m4 macro and I just fill a variable with the files, the macro defines the install rules from that variable

Reply:
Ah. If they don't specify it in the pkg-config file or via some other easily accessible way I'm afraid we'll have to stick with this. In _theory_ one can execute m4 macros in meson (take a look at NM for that) but let's _please_ not do that